sass/_Breadcrumb.scss

```diff
@@ -46,7 +46,7 @@
   margin: 5px 0px 5px 0px;
 }
 
-@include mediaSelector(800) {
+@include smallScreenWidth() {
```

Comment:
This one looks like it should be `mediumScreenWidth`.

Reply:
Good catch, but I did change it to small screen width on purpose :) 

There was a visual bug for screen size between 800-480 px